### ISSUE TITLE
[Bug]: First 3 digits of Year Taken as Series Number

### ISSUE THREAD

**Issue by amoskyler** [advplyr/audiobookshelf#695]
### Describe the issue

When a series folder is present, volume numbers are parsed as the first 3 digits of year, rather than the volume number.

Example:

```
Robert Jordan /
-| Wheel of Time /
---| Vol 1 - 1990 - The Eye of the World {Kate Reading & Michael Kramer} /
-----| book.mb4
```

The metadata for the above results in a series/volume # of `Wheel of Time#199`.

It should be `Wheel of Time#1`

### Steps to reproduce the issue

1. Create a folder structure like above
2. Scan library
3. Observe scanned metadata for the book

### Audiobookshelf version

2.0.20

### How are you running audiobookshelf?

Docker

**Comment by advplyr** [comment]
Can you try to reproduce this in the latest release 2.0.21?
@cassieesposito fixed something with this and maybe able to add more context.

**Comment by cassieesposito** [comment]
I found and fixed this as a part of the fix to Issue #691. This should be fixed in 2.0.21. Please let me know if it isn't.
